source/renderer/app/api/network/types.js

```diff
@@ -14,6 +14,11 @@ export type FutureEpoch = {
   epochStart: string,
 };
 
+export type ClockOffset = {
+  quantity: number,
+  unit: string,
```

Comment:
@yakovkaravelov please improve the type for unit here and set it to `microsecond` 🙏